call-home/src/common/constants.rs

```diff
@@ -1 +1,7 @@
 pub(crate) const PRODUCT: &str = "Mayastor";
+pub(crate) const DATA_DIR_PATH: &str = "/encryption_dir";
+pub(crate) const ENCRYPTION_KEY_FILEPATH: &str = "/key/";
+pub(crate) const SWARM_API_ENDPOINT: &str = "";
```

Comment:
```suggestion
pub(crate) const RECEIVER_API_ENDPOINT: &str = "";
```

Reply:
All things Swarm is now known as Receiver.